build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
 
     //Unit testing framework.
     testCompile 'junit:junit:4.12'
+    //testCompile "org.mockito:mockito-core:2.+"
```

Comment:
OK, we can leave that in there - can we just add something to say "needed for mockito" or something?